packages/commonjs-shim/src/utils.ts

```diff
@@ -0,0 +1,35 @@
+import MagicString from 'magic-string';
+
+// Shim __dirname, __filename and require
+import { CJSShim, CJSyntaxRe, ESM_STATIC_IMPORT_RE } from './constants';
```

Comment:
let's pick one format for these const names please. you'll see the format `export const reThingBob` throughout the repo as well.

Reply:
Since we are dealing with constants and not variables, the first letter should be capitalized in any case. done.